tests/integration/jit/tracing/test_computations.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Transaction event generated for heavy computation.  No errors or failures generated despite creating spans for a huge number of calls.
+*/
+
+/*SKIPIF
+<?php
+
+require('skipif.inc');
+
+*/
+
+/*INI
+newrelic.distributed_tracing_enabled=1
+newrelic.transaction_tracer.threshold = 0
+error_reporting = E_ALL
+opcache.enable=1
+opcache.enable_cli=1
+opcache.file_update_protection=0
+opcache.jit_buffer_size=32M
+opcache.jit=function
```

Comment:
I think all of these in the `tracing` directory need to have `opcache.jit` set to `tracing` instead of `function`.